packages/shared/lib/profile.ts

```diff
@@ -12,8 +18,13 @@ import { AvailableExchangeRates } from './typings/currency'
 import type { WalletAccount } from './typings/wallet'
 import { getOfficialNetworkConfig } from './network'
 import { NetworkConfig, NetworkType } from './typings/network'
+import { account } from './typings'
 
 export const activeProfileId = persistent<string | null>('activeProfileId', null)
+export interface ProfileAccount {
```

Comment:
Should separate interfaces into type files. Will be covered by the big refactor.